src/views/Profile/AdminTable/AdminFilter.tsx

```diff
@@ -51,6 +50,30 @@ export const AdminFilter: React.FC<IMaterialsFilter> = ({
 
   const classes = useStyles();
   const { t } = useTranslation();
+  const [open, setOpen] = useState(false);
+  const selectRef = useRef(null);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      if (open) {
```

Comment:
but do we really need this check? I guess, calling `setOpen(false)` will work fine even if dropdown is already closed. No?